lib/ddtrace/contrib/rails/utils.rb

```diff
@@ -33,6 +33,8 @@ def self.normalize_vendor(vendor)
             'sqlite'
           when 'postgresql'
             'postgres'
+          when 'mysql2'
+            'mysql'
```

Comment:
I wasn't sure if you would want to differentiate between mysql and mysql2 adapters. If you do, can probably remove this line, otherwise this should group them together.